From ca387a733c4207f4735347f773246d05f46d3da5 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Fri, 9 Oct 2026 02:44:50 +0000
Subject: [PATCH] Refresh the guest package lock

adwaita-cursors 50.0-1 -> 51.0-1
adwaita-icon-theme 50.0-1 -> 51.0-1
at-spi2-core 2.60.7-1 -> 2.62.0.1-1
ca-certificates-mozilla 3.130-1 -> 3.131-1
dconf 0.49.0-1 -> 51.0-1
dkms 3.4.3-2 -> 3.4.4-1
expat 2.8.5-1 -> 2.9.0-1
fcitx5-qt 5.1.16-1 -> 5.1.16-2
gdk-pixbuf2 2.44.7-1 -> 2.44.8-1
glib-networking 1:2.80.1-1 -> 1:2.90.0-1
glib2 2.88.3-1 -> 2.90.1-1
glycin 2.1.5-2 -> 2.2.2-1
glycin-gtk4 2.1.5-2 -> 2.2.2-1
gnome-desktop-4 1:44.5-1 -> 1:51.0-1
gnome-desktop-common 1:44.5-1 -> 1:51.0-1
gnome-keyring 1:50.0-1 -> 1:51.1-1
groff 1.24.1-1 -> 1.24.2-1
gsettings-desktop-schemas 50.1-1 -> 51.0-1
gsettings-system-schemas 50.1-1 -> 51.0-1
gst-plugins-bad-libs 1.28.7-2 -> 1.28.7-3
gst-plugins-base-libs 1.28.7-2 -> 1.28.7-3
gstreamer 1.28.7-2 -> 1.28.7-3
gtk-update-icon-cache 1:4.22.5-1 -> 1:4.24.1-1
gtk3 1:3.24.52-1 -> 1:3.24.52-2
gtk4 1:4.22.5-1 -> 1:4.24.1-1
gvfs 1.60.3-3 -> 1.62.0-3
inotify-tools 4.26.268-1 -> 4.26.270-1
libadwaita 1:1.9.4-1 -> 1:1.10.0-1
libcups 2:2.4.19-1 -> 2:2.4.20-1
libevdev 1.13.7-1 -> 1.14.0-1
libnautilus-extension 50.3.1-1 -> 51.0.1-2
librsvg 2:2.62.4-1 -> 2:2.63.2-1
libsoup3 3.6.6-2 -> 3.8.0-1
libsysprof-capture 50.0-6 -> 51.0-1
libx11 1.8.13-1 -> 1.8.13-2
linux 7.2.8.arch1-2 -> 7.2.9.arch1-1
linux-headers 7.2.8.arch1-2 -> 7.2.9.arch1-1
localsearch 3.11.2-1 -> 3.12.0-1
nautilus 50.3.1-1 -> 51.0.1-2
nss 3.130-1 -> 3.131-1
openssh 10.5p1-1 -> 10.6p1-1
python-gobject 3.56.3-1 -> 3.58.0-1
qt6-base 6.11.2-3 -> 6.12.0-2
qt6-declarative 6.11.2-2 -> 6.12.0-1
qt6-multimedia 6.11.2-1 -> 6.12.0-1
qt6-multimedia-ffmpeg 6.11.2-1 -> 6.12.0-1
qt6-svg 6.11.2-1 -> 6.12.0-1
qt6-translations 6.11.2-1 -> 6.12.0-1
qt6-wayland 6.11.2-1 -> 6.12.0-1
quickshell 0.3.1-1 -> 0.3.2-1
tinysparql 3.11.1-1 -> 3.12.0-1
xorg-server 21.1.24-1 -> 21.1.25-1
xorg-server-common 21.1.24-1 -> 21.1.25-1
xorg-xwayland 24.1.13-1 -> 24.1.14-1
xorgproto 2025.1-1 -> 2026.1-1
gd 2.3.3-9 (new)
ghostscript 10.08.0-1 (new)
graphviz 16.1.0-1 (new)
gsfonts 20200910-6 (new)
gts 0.7.6.121130-5 (new)
ijs 0.35-7 (new)
jbig2dec 0.20-2 (new)
libavif 1.4.2-1 (new)
libidn 1.44-1 (new)
libmediainfo 26.10-1 (new)
libmms 0.6.4-6 (new)
libpaper 2.2.8-1 (new)
libxpm 3.5.19-1 (new)
libyuv r2921+644251f25-1 (new)
libzen 0.4.41-3 (new)
ministream 0.99.1-1 (new)
netpbm 10.86.51-1 (new)
poppler-data 0.4.12-2 (new)
tinyxml2 11.0.0-2 (new)
appstream (removed)
cfitsio (removed)
hdf5 (removed)
hwloc (removed)
imath (removed)
libaec (removed)
libblake3 (removed)
libcgif (removed)
libfyaml (removed)
libimagequant (removed)
libmatio (removed)
libraw (removed)
libvips (removed)
libxmlb (removed)
onetbb (removed)
openexr (removed)
openjph (removed)
---
 guest/packages.lock.json | 146 ++++++++++++++++++++-------------------
 1 file changed, 74 insertions(+), 72 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 24b7ee3..0b083cc 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -3,19 +3,18 @@
   "packages": {
     "abseil-cpp": "20260817.0-2",
     "acl": "2.4.0-1",
-    "adwaita-cursors": "50.0-1",
+    "adwaita-cursors": "51.0-1",
     "adwaita-fonts": "51.0-2",
-    "adwaita-icon-theme": "50.0-1",
+    "adwaita-icon-theme": "51.0-1",
     "adwaita-icon-theme-legacy": "46.2-3",
     "alsa-card-profiles": "1:1.6.9-1",
     "alsa-lib": "1.2.16.1-1",
     "alsa-topology-conf": "1.2.5.1-4",
     "alsa-ucm-conf": "1.2.16.1-1",
     "aom": "3.15.1-1",
-    "appstream": "1.2.1-1",
     "aquamarine": "0.15.1-1",
     "archlinux-keyring": "20260909-1",
-    "at-spi2-core": "2.60.7-1",
+    "at-spi2-core": "2.62.0.1-1",
     "attr": "2.6.0-1",
     "audit": "4.2.1-1",
     "autoconf": "2.73-1",
@@ -35,10 +34,9 @@
     "bubblewrap": "0.13.0-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.130-1",
+    "ca-certificates-mozilla": "3.131-1",
     "ca-certificates-utils": "20240618-1",
     "cairo": "1.18.6-1",
-    "cfitsio": "1:4.7.0-1",
     "chromium": "153.0.8010.52-1",
     "clang": "23.1.1-1",
     "compiler-rt": "23.1.1-1",
@@ -53,13 +51,13 @@
     "dbus-broker": "37-3",
     "dbus-broker-units": "37-3",
     "dbus-units": "37-3",
-    "dconf": "0.49.0-1",
+    "dconf": "51.0-1",
     "debugedit": "5.3-2",
     "default-cursors": "3-1",
     "desktop-file-utils": "0.28-1",
     "device-mapper": "2.03.43-2",
     "diffutils": "3.12-2",
-    "dkms": "3.4.3-2",
+    "dkms": "3.4.4-1",
     "double-conversion": "3.4.0-1",
     "dua-cli": "2.45.1-1",
     "duktape": "2.7.0-7",
@@ -68,7 +66,7 @@
     "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
-    "expat": "2.8.5-1",
+    "expat": "2.9.0-1",
     "eza": "0.23.5-2",
     "fakeroot": "1:1.37.2-3",
     "fastfetch": "2.69.0-1",
@@ -76,7 +74,7 @@
     "fcitx5": "5.1.23-1",
     "fcitx5-chewing": "5.1.13-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.16-1",
+    "fcitx5-qt": "5.1.16-2",
     "fd": "10.5.0-3",
     "ffmpeg": "2:9.0.2-2",
     "fftw": "3.3.11-1",
@@ -98,27 +96,29 @@
     "gcc-libs": "16.2.1+r23+gd564253eb6c8-1",
     "gcr": "3.41.2-2",
     "gcr-4": "4.4.1-1",
+    "gd": "2.3.3-9",
     "gdb": "18.1-1",
     "gdb-common": "18.1-1",
     "gdbm": "1.26-2",
-    "gdk-pixbuf2": "2.44.7-1",
+    "gdk-pixbuf2": "2.44.8-1",
     "gettext": "1.0-2",
     "gexiv2": "0.16.2-2",
     "gexiv2-common": "0.16.2-2",
+    "ghostscript": "10.08.0-1",
     "giflib": "6.1.3-2",
     "git": "2.56.0-1",
-    "glib-networking": "1:2.80.1-1",
-    "glib2": "2.88.3-1",
+    "glib-networking": "1:2.90.0-1",
+    "glib2": "2.90.1-1",
     "glibc": "2.44+r50+g1848099f063e-1",
     "glslang": "1:1.4.363.0-1",
     "glu": "9.0.3-3",
-    "glycin": "2.1.5-2",
-    "glycin-gtk4": "2.1.5-2",
+    "glycin": "2.2.2-1",
+    "glycin-gtk4": "2.2.2-1",
     "gmp": "6.3.0-3",
     "gnome-autoar": "0.5.2-1",
-    "gnome-desktop-4": "1:44.5-1",
-    "gnome-desktop-common": "1:44.5-1",
-    "gnome-keyring": "1:50.0-1",
+    "gnome-desktop-4": "1:51.0-1",
+    "gnome-desktop-common": "1:51.0-1",
+    "gnome-keyring": "1:51.1-1",
     "gnulib-l10n": "20241231-1",
     "gnupg": "2.4.9-3",
     "gnutls": "3.8.13-3",
@@ -130,36 +130,37 @@
     "gpu-screen-recorder": "6.1.3-1",
     "graphene": "1.10.8-2",
     "graphite": "1:1.3.15-1",
+    "graphviz": "16.1.0-1",
     "grep": "3.12-2",
     "grim": "1.5.0-2",
-    "groff": "1.24.1-1",
-    "gsettings-desktop-schemas": "50.1-1",
-    "gsettings-system-schemas": "50.1-1",
+    "groff": "1.24.2-1",
+    "gsettings-desktop-schemas": "51.0-1",
+    "gsettings-system-schemas": "51.0-1",
+    "gsfonts": "20200910-6",
     "gsm": "1.0.24-1",
     "gssdp": "1.6.6-1",
-    "gst-plugins-bad-libs": "1.28.7-2",
-    "gst-plugins-base-libs": "1.28.7-2",
-    "gstreamer": "1.28.7-2",
+    "gst-plugins-bad-libs": "1.28.7-3",
+    "gst-plugins-base-libs": "1.28.7-3",
+    "gstreamer": "1.28.7-3",
     "gtest": "1.18.0-1",
-    "gtk-update-icon-cache": "1:4.22.5-1",
-    "gtk3": "1:3.24.52-1",
-    "gtk4": "1:4.22.5-1",
+    "gtk-update-icon-cache": "1:4.24.1-1",
+    "gtk3": "1:3.24.52-2",
+    "gtk4": "1:4.24.1-1",
     "gtk4-layer-shell": "1.3.0-1",
+    "gts": "0.7.6.121130-5",
     "guile": "3.0.11-1",
     "gum": "2.0.2-1",
     "gupnp": "1:1.6.10-1",
     "gupnp-dlna": "0.12.0-5",
     "gupnp-igd": "1.6.0-2",
-    "gvfs": "1.60.3-3",
+    "gvfs": "1.62.0-3",
     "gzip": "1.15-1",
     "harfbuzz": "14.6.0-1",
-    "hdf5": "2.2.0-1",
     "herdr": "0.8.2-1",
     "hicolor-icon-theme": "0.18-1",
     "hidapi": "0.15.0-1",
     "highway": "1.4.0-1",
     "hwdata": "0.412-1",
-    "hwloc": "2.14.0-1",
     "hyprcursor": "0.1.13-7",
     "hyprgraphics": "0.5.1-4",
     "hypridle": "0.1.8-2",
@@ -174,17 +175,18 @@
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
+    "ijs": "0.35-7",
     "imagemagick": "7.1.2.32-1",
-    "imath": "3.2.3-1",
     "imv": "5.0.1-2",
     "iniparser": "4.3.2-1",
-    "inotify-tools": "4.26.268-1",
+    "inotify-tools": "4.26.270-1",
     "iproute2": "7.2.0-1",
     "iptables": "1:1.8.13-1",
     "iputils": "20250605-1",
     "iso-codes": "4.20.1-1",
     "jack2": "1.9.22-2",
     "jansson": "2.15.1-1",
+    "jbig2dec": "0.20-2",
     "jbigkit": "2.1-8",
     "jemalloc": "1:5.4.0-1",
     "jq": "1.8.2-1",
@@ -200,8 +202,7 @@
     "leancrypto": "1.9.1-1",
     "leptonica": "1.87.0-2",
     "less": "1:710-1",
-    "libadwaita": "1:1.9.4-1",
-    "libaec": "1.1.7-1",
+    "libadwaita": "1:1.10.0-1",
     "libarchive": "3.8.9-1",
     "libasan": "16.2.1+r23+gd564253eb6c8-1",
     "libass": "0.17.5-1",
@@ -210,8 +211,8 @@
     "libatasmart": "0.19-8",
     "libatomic": "16.2.1+r23+gd564253eb6c8-1",
     "libavc1394": "0.5.4-7",
+    "libavif": "1.4.2-1",
     "libb2": "0.98.1-3",
-    "libblake3": "1.8.7-1",
     "libblockdev": "3.5.0-2",
     "libblockdev-crypto": "3.5.0-2",
     "libblockdev-fs": "3.5.0-2",
@@ -230,12 +231,11 @@
     "libcap-ng": "0.9.6-1",
     "libcdio": "2.4.0-1",
     "libcdio-paranoia": "10.2+2.0.2-3",
-    "libcgif": "0.5.4-1",
     "libchewing": "0.13.1-1",
     "libcloudproviders": "0.4.1-1",
     "libcolord": "1.4.8-1",
     "libcue": "2.3.0-2",
-    "libcups": "2:2.4.19-1",
+    "libcups": "2:2.4.20-1",
     "libdaemon": "0.14-6",
     "libdatrie": "0.2.14-1",
     "libde265": "1.1.3-1",
@@ -252,14 +252,13 @@
     "libei": "1.6.0-1",
     "libelf": "0.196-1",
     "libepoxy": "1.5.10-3",
-    "libevdev": "1.13.7-1",
+    "libevdev": "1.14.0-1",
     "libevent": "2.1.13-2",
     "libexif": "0.6.26-1",
     "libfdk-aac": "2.0.3-2",
     "libffi": "3.8.0-1",
     "libfontenc": "1.1.9-1",
     "libfreeaptx": "0.2.2-1",
-    "libfyaml": "0.9.6-3",
     "libgcc": "16.2.1+r23+gd564253eb6c8-1",
     "libgcrypt": "1.12.4-1",
     "libgfortran": "16.2.1+r23+gd564253eb6c8-1",
@@ -274,9 +273,9 @@
     "libheif": "1.23.6-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
     "libice": "1.1.2-1",
+    "libidn": "1.44-1",
     "libidn2": "2.3.8-1",
     "libiec61883": "1.2.0-9",
-    "libimagequant": "4.4.1-2",
     "libimobiledevice": "1.4.0-2",
     "libimobiledevice-glue": "1.3.2-1",
     "libinih": "62-2",
@@ -293,14 +292,15 @@
     "liblsan": "16.2.1+r23+gd564253eb6c8-1",
     "libluv": "1.52.1-1",
     "libmakepkg-dropins": "20-2",
-    "libmatio": "1.5.30-1",
     "libmd": "1.3.0-1",
+    "libmediainfo": "26.10-1",
     "libmm-glib": "1.24.2-1",
+    "libmms": "0.6.4-6",
     "libmnl": "1.0.5-2",
     "libmodplug": "0.8.9.0-7",
     "libmpc": "1.4.1-1",
     "libmysofa": "1.3.5-1",
-    "libnautilus-extension": "50.3.1-1",
+    "libnautilus-extension": "51.0.1-2",
     "libndp": "1.9-1",
     "libnetfilter_conntrack": "1.1.1-1",
     "libnewt": "0.52.25-2",
@@ -322,6 +322,7 @@
     "libopenmpt": "0.8.9-1",
     "libosinfo": "1.12.0-3",
     "libp11-kit": "0.26.5-1",
+    "libpaper": "2.2.8-1",
     "libpcap": "1.11.0-1",
     "libpciaccess": "0.19-1",
     "libpgm": "5.3.128-4",
@@ -337,9 +338,8 @@
     "libpulse": "17.0+r98+gb096704c0-1",
     "libquadmath": "16.2.1+r23+gd564253eb6c8-1",
     "libraqm": "0.11.0-1",
-    "libraw": "0.22.2-1",
     "libraw1394": "2.1.2-4",
-    "librsvg": "2:2.62.4-1",
+    "librsvg": "2:2.63.2-1",
     "libsamplerate": "0.2.2-3",
     "libsasl": "2.1.28-5",
     "libseccomp": "2.6.0-1",
@@ -348,13 +348,13 @@
     "libsm": "1.2.6-1",
     "libsndfile": "1.2.2-4",
     "libsodium": "1.0.22-1",
-    "libsoup3": "3.6.6-2",
+    "libsoup3": "3.8.0-1",
     "libsoxr": "0.1.3-5",
     "libssh": "0.12.2-1",
     "libssh2": "1.11.1-7",
     "libstdc++": "16.2.1+r23+gd564253eb6c8-1",
     "libstemmer": "3.1.1-1",
-    "libsysprof-capture": "50.0-6",
+    "libsysprof-capture": "51.0-1",
     "libtasn1": "4.21.0-1",
     "libtatsu": "1.0.5-1",
     "libteam": "1.32-3",
@@ -377,7 +377,6 @@
     "libva": "2.24.1-1",
     "libvdpau": "1.5-4",
     "libverto": "0.3.2-6",
-    "libvips": "8.18.7-1",
     "libvorbis": "1.3.7-4",
     "libvpl": "2.17.0-1",
     "libvpx": "1.17.0-1",
@@ -385,7 +384,7 @@
     "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
     "libwireplumber": "0.5.18-1",
-    "libx11": "1.8.13-1",
+    "libx11": "1.8.13-2",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
     "libxcomposite": "0.4.7-1",
@@ -404,8 +403,8 @@
     "libxkbcommon-x11": "1.13.2-1",
     "libxkbfile": "1.2.0-1",
     "libxml2": "2.15.4-1",
-    "libxmlb": "0.3.29-1",
     "libxmu": "1.3.1-1",
+    "libxpm": "3.5.19-1",
     "libxpresent": "1.0.2-1",
     "libxrandr": "1.5.5-1",
     "libxrender": "0.9.12-1",
@@ -417,17 +416,19 @@
     "libxv": "1.0.13-1",
     "libxxf86vm": "1.1.7-1",
     "libyaml": "0.2.5-3",
+    "libyuv": "r2921+644251f25-1",
+    "libzen": "0.4.41-3",
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.8.arch1-2",
+    "linux": "7.2.9.arch1-1",
     "linux-api-headers": "7.2-1",
-    "linux-headers": "7.2.8.arch1-2",
+    "linux-headers": "7.2.9.arch1-1",
     "llhttp": "9.3.1-1",
     "llvm-libs": "23.1.1-1",
     "lm_sensors": "1:3.6.2-1",
     "lmdb": "0.9.35-1",
-    "localsearch": "3.11.2-1",
+    "localsearch": "3.12.0-1",
     "lua": "5.5.1-1",
     "lua51-lpeg": "1.1.0-5",
     "lua54": "5.4.9-1",
@@ -441,6 +442,7 @@
     "md4c": "0.6.0-2",
     "mdadm": "4.6-2",
     "mesa": "1:26.2.4-1",
+    "ministream": "0.99.1-1",
     "minizip": "1:1.3.2-3",
     "mkinitcpio": "42.2-1",
     "mkinitcpio-busybox": "1.36.1-1",
@@ -453,9 +455,10 @@
     "mtdev": "1.1.7-1",
     "mujs": "1.3.10-1",
     "muparser": "2.3.5-2",
-    "nautilus": "50.3.1-1",
+    "nautilus": "51.0.1-2",
     "ncurses": "6.6-2",
     "neovim": "0.12.5-1",
+    "netpbm": "10.86.51-1",
     "nettle": "4.0-1",
     "networkmanager": "1.58.1-2",
     "nftables": "1:1.1.7-3",
@@ -464,7 +467,7 @@
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.130-1",
+    "nss": "3.131-1",
     "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
     "omacalc": "0.2.2-1",
@@ -472,15 +475,12 @@
     "omarchy-keyring": "20251027-1",
     "omarchy-nvim": "2026.8.13-1",
     "omawrite": "0.5.0-1",
-    "onetbb": "2023.1.0-1",
     "oniguruma": "6.9.10-1",
     "openal": "1.25.2-2",
     "opencore-amr": "0.1.6-2",
-    "openexr": "3.5.2-1",
     "openh264": "2.6.0-2",
     "openjpeg2": "2.5.4-1",
-    "openjph": "0.32.0-1",
-    "openssh": "10.5p1-1",
+    "openssh": "10.6p1-1",
     "openssl": "3.6.5-1",
     "opus": "1.6.1-1",
     "orc": "0.4.44-1",
@@ -515,6 +515,7 @@
     "plocate": "1.1.25-1",
     "polkit": "127-3",
     "poppler": "26.08.0-1",
+    "poppler-data": "0.4.12-2",
     "poppler-glib": "26.08.0-1",
     "popt": "1.19-2",
     "portaudio": "1:19.7.0-4",
@@ -524,19 +525,19 @@
     "python": "3.14.7-1",
     "python-dbus": "1.5.0-1",
     "python-docopt": "0.6.2-15",
-    "python-gobject": "3.56.3-1",
+    "python-gobject": "3.58.0-1",
     "python-pyxdg": "0.28-7",
     "python-yaml": "6.0.3-2",
     "qoi": "r350.4461cc3-1",
     "qrencode": "4.1.1-4",
-    "qt6-base": "6.11.2-3",
-    "qt6-declarative": "6.11.2-2",
-    "qt6-multimedia": "6.11.2-1",
-    "qt6-multimedia-ffmpeg": "6.11.2-1",
-    "qt6-svg": "6.11.2-1",
-    "qt6-translations": "6.11.2-1",
-    "qt6-wayland": "6.11.2-1",
-    "quickshell": "0.3.1-1",
+    "qt6-base": "6.12.0-2",
+    "qt6-declarative": "6.12.0-1",
+    "qt6-multimedia": "6.12.0-1",
+    "qt6-multimedia-ffmpeg": "6.12.0-1",
+    "qt6-svg": "6.12.0-1",
+    "qt6-translations": "6.12.0-1",
+    "qt6-wayland": "6.12.0-1",
+    "quickshell": "0.3.2-1",
     "rav1e": "0.8.1-3",
     "re2": "2:2025.11.05-6",
     "readline": "8.3.6-1",
@@ -580,7 +581,8 @@
     "tesseract-data-eng": "2:4.1.0-5",
     "tesseract-data-osd": "2:4.1.0-5",
     "texinfo": "7.3-1",
-    "tinysparql": "3.11.1-1",
+    "tinysparql": "3.12.0-1",
+    "tinyxml2": "11.0.0-2",
     "tmux": "3.7_c-1",
     "tomlplusplus": "3.4.0-2",
     "totem-pl-parser": "3.26.7-1",
@@ -646,14 +648,14 @@
     "xf86-input-libinput": "1.5.0-1",
     "xkeyboard-config": "2.48-1",
     "xorg-fonts-encodings": "1.1.0-2",
-    "xorg-server": "21.1.24-1",
-    "xorg-server-common": "21.1.24-1",
+    "xorg-server": "21.1.25-1",
+    "xorg-server-common": "21.1.25-1",
     "xorg-setxkbmap": "1.3.5-1",
     "xorg-xauth": "1.1.5-1",
     "xorg-xkbcomp": "1.5.0-1",
     "xorg-xprop": "1.2.8-1",
-    "xorg-xwayland": "24.1.13-1",
-    "xorgproto": "2025.1-1",
+    "xorg-xwayland": "24.1.14-1",
+    "xorgproto": "2026.1-1",
     "xvidcore": "1.3.7-4",
     "xxhash": "0.8.4-1",
     "xz": "5.8.4-1",
-- 
2.55.0

